source/administration/config-settings.rst

```diff
@@ -4518,6 +4552,57 @@ Options for printing Elasticsearch trace errors.  Accepts ``error``, ``all``, or
 | This feature's ``config.json`` setting is ``"Trace": ""`` with string input.                          |
 +-------------------------------------------------------------------------------------------------------+
 
+Bleve Settings (Experimental)
+~~~~~~~~~~~~~~~
+
+Index Dir
+^^^^^^^^^^^^^^^^^^^^^^^
+Directory path to use for storing bleve indexes.
+
++-----------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"IndexDir "`` with string input.                              |
++-----------------------------------------------------------------------------------------------------------+
+
+Enable Indexing
+^^^^^^^^^^^^^^^^^^^^^^^
+**True:** The indexing of new posts occurs automatically. Search queries will use database search until "Enable Bleve for search queries" is enabled.
```

Comment:
nit: rewording to avoid referencing database as searches could be using Elasticsearch or other search engine in the future

```suggestion
**True:** The indexing of new posts occurs automatically. Search queries will not use bleve search until "Enable Bleve for search queries" is enabled.
```